pkg/challenge/challenge.go

```diff
@@ -64,11 +64,20 @@ func (o *OuterChallenge) Respond(signer crypto.Signer, responseData []byte) ([]b
 		return nil, fmt.Errorf("marshalling public signing key to pem: %w", err)
 	}
 
+	var pubSigningKeyPem2 []byte
+	if signer2 != nil {
+		pubSigningKeyPem2, err = echelper.PublicEcdsaKeyToPem(signer2.Public().(*ecdsa.PublicKey))
```

Comment:
We're _also_ moving away from pem, right?

Reply:
updated everything to use DER, but added some tests to make sure we also handle PEM